doc/source/installation/yum_repo.rst

```diff
@@ -78,13 +78,16 @@ Installing |Percona Server| from Percona ``yum`` repository
 
 1. Install the Percona repository 
    
-   You can install Percona yum repository by running the following command as a ``root`` user or with sudo:
+By default, Percona Server installs and updates creates a mysqld.cnf as
```

Comment:
a bit strange `installs and updates creates a mysqld.cnf` shouldn't it be `installs and updates a mysqld.cnf` or something similar?

Reply:
Rewrote the paragraph.